google-api-client/src/main/java/com/google/api/client/googleapis/json/GoogleJsonResponseException.java

```diff
@@ -141,6 +141,10 @@ public static GoogleJsonResponseException from(JsonFactory jsonFactory, HttpResp
     }
     // message
     StringBuilder message = HttpResponseException.computeMessageBuffer(response);
```

Comment:
Maybe I should make this change in the [`HttpResponseException. computeMessageBuffer`](https://github.com/googleapis/google-http-java-client/blob/027c227e558164f77be204152fb47023850b543f/google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java#L274) method in [google-http-java-client](https://github.com/googleapis/google-http-java-client) instead? @elharo wdyt?

Reply:
That could work.

Reply:
Send PR: https://github.com/googleapis/google-http-java-client/pull/1002

Reply:
Re-purposed this PR for upgrading to newer HTTP client (when it will be released) that has request method in URL in the exception message.